plugins/repoquery.py

```diff
@@ -226,11 +226,9 @@ class RepoQueryCommand(dnf.cli.Command):
 
     @staticmethod
     def by_dep(sack, pattern, query, dep):
-        try:
-            reldep = hawkey.Reldep(sack, pattern)
-        except hawkey.ValueException:
-            return query.filter(empty=True)
-        kwarg = {dep: reldep}
+        if dnf.util.is_glob_pattern(pattern):
+            dep += '__glob'
+        kwarg = {dep: pattern}
         return query.filter(**kwarg)
 
```

Comment:
> BTW, isn't hawkey.Reldep faster than filter?

It's the same. (Exactly same string to reldep conversion is the first thing which happen in the filter.)